common/src/main/java/com/wynntils/models/containers/type/SearchableContainerProperty.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.containers.type;
+
+public interface SearchableContainerProperty extends ScrollableContainerProperty {
+    ContainerBounds getBounds();
+
+    boolean supportsAdvancedSearch();
```

Comment:
Did we ever have a way to disable the advanced search with a config? If yes, do you know if that works? (If we have no such config, ignore this :)).

Reply:
I don't think so, when the advanced search was added it just replaced the original basic search